packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -193,21 +193,25 @@ export function ArticleEditor({id}: ArticleEditorProps) {
   }
 
   // Reads title and lead from the first block and saves them in variables
-  function syncBlocksWithMeta() {
-    const titleBlock = blocks.map(block => {
-      if (block.type === 'title') {
-        return block.value
-      } else {
-        return false
-      }
-    })
+  function syncFirstTitleBlockWithMetadata() {
+    interface titleBlockToSync {
+      title: string
+      lead: string
+    }
+
+    if ((metadata.title === '' || metadata.lead === '') && blocks.length > 0) {
```

Comment:
I would use `&&` here. If the user open the metadata before and added data to either `title` or `lead` it might be that they will add the other property later on.